### PR TITLE
Update to build 4.2.3 and 5.3.0

### PR DESCRIPTION
**Pull request by MariusRumpf**
Somebody will have to take care of `Makefile` and `.drone.yml` since I can't make that work, some inspiration on how to make it work might be found in the `test.sh`
- Integrated `4.2.3` and `5.3.0` instead of only one build
- I moved over from `build-essential git libssl-dev python ca-certificates curl` to using the same dependencies as the original x86 docker image (lists from buildpacks-deps:jessie and buildpacks-deps:wheezy)
- Building can be tested with `test.sh`
- I successfully build and tested the images on a Raspberry Pi 2 and have been using it for my own node app
- `0.10` and `0.12` builds for linux-armv6l are not available by the node foundation

### CODE REVIEW

**Comment by StefanScherer** [comment]
@MariusRumpf This is awesome! Yes, we have to build both of the stable versions. We have to setup our Drone CI to give better feedback for such PR's in the future.

File: 4.2/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,74 @@
+FROM resin/rpi-raspbian:jessie
+
+# Install common build dependencies
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    ca-certificates \
```
Comment (StefanScherer) [inline 4.2/Dockerfile:5]: So this is the list from buildpacks-deps:jessie, right?

File: test.sh [hunk 2]
```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
```
Comment (StefanScherer) [inline test.sh]: That's a better way than to try this in a Makefile. I really like this approach. Could you please rename the script to `build.sh`, so we can add a `test.sh` to do some tests later with the images.

I think of some CI scripts
- build.sh
- test.sh
- deploy.sh

I will care of the last two scripts ;-)

File: 4.2/Dockerfile (hunk 1 above)
Comment (MariusRumpf) [inline 4.2/Dockerfile:5]: Thats correct.

**Comment by StefanScherer** [comment]
Thanks!

**Pull request merged**